Bukkit/src/main/java/com/plotsquared/bukkit/BukkitPlatform.java

```diff
@@ -18,57 +18,28 @@
  */
 package com.plotsquared.bukkit;
 
-import com.google.inject.Guice;
-import com.google.inject.Inject;
-import com.google.inject.Injector;
-import com.google.inject.Key;
-import com.google.inject.Singleton;
-import com.google.inject.Stage;
-import com.google.inject.TypeLiteral;
+import com.google.inject.*;
```

Comment:
Don't use asterisk imports. Please configure your IDE to respect the `.editorconfig` file placed in the root directory.

Reply:
My ide dont respect the settings 😂 Idk why but all the default settings